tests/robot-tests/scripts/install_webdriver.py

```diff
@@ -0,0 +1,20 @@
+import pyderman
+import platform
+from pathlib import Path
+import os
+
+
+def install_webdriver(version: str) -> None:
+    chromedriver_filename = 'chromedriver.exe' if platform.system() == "Windows" else 'chromedriver'
+    pyderman.install(file_directory='../webdriver/',
+                     filename=chromedriver_filename,
+                     verbose=True,
+                     chmod=True,
+                     overwrite=False,
+                     version=version)
+
+    os.environ["PATH"] += os.pathsep + str(Path('../webdriver').absolute())
```

Comment:
Maybe to alleviate potential pathing issues, we could change this to `get_webdriver.py`, which we would also use in `get_identity_info` to avoid having to set the path there as well